exporter/zipkin/zipkin.go

```diff
@@ -50,6 +54,36 @@ func NewExporter(reporter reporter.Reporter, localEndpoint *model.Endpoint) *Exp
 	}
 }
 
+type Options struct {
+	// EndpointHostPort is the host:port combination for the Zipkin service
+	// e.g. 192.168.0.2:8080
+	EndpointHostPort string
+
+	// ReporterURI is the HTTP endpoint to which spans will be sent
+	// e.g. http://localhost:9411/api/v2/spans
+	ReporterURI string
```

Comment:
Is this really an option, or is it required?

Reply:
It is required, it is marked optional, but sure, I'll comment on it as required.

Reply:
Why not make it an argument. Being an option suggests that it's optional.

Reply:
You raise a great point. However, I am following suit with the style of our other exporters
* https://godoc.org/go.opencensus.io/exporter/stackdriver#Options
* https://godoc.org/go.opencensus.io/exporter/prometheus#Options
Plus it is a lot easier to deal with growing arguments for initialization, while using a struct in a backwards compatible way, than with breaking the APIs everytime we need to add an argument.